ynr/settings/constants/needs_review.py

```diff
@@ -1,3 +1,7 @@
+from datetime import datetime, timedelta
+
 # This is the number of the first edits of a user that are considered
 # to need review.
 NEEDS_REVIEW_FIRST_EDITS = 3
+NEEDS_REVIEW_MAX_REVERTS = 2
+NEEDS_REVIEW_TIME_PERIOD = datetime.now() - timedelta(hours=24)
```

Comment:
This is a subtle problem that's worth knowing about:

Because this is imported in settings, the code is evaluated when Django starts up, rather than each time the variable is called. 

This means that the value of `NEEDS_REVIEW_TIME_PERIOD` will be 24 hours before the server started (e.g, if you started `runserver` it would be set to 24 hours before), but then it would never change, because it's never re-evaluated.

The solution is to put the delta (24) settings (`NEEDS_REVIEW_TIME_PERIOD = 24`). Then in the `needs_review` method add `updated__gt=datetime.now() - timedelta(settings.NEEDS_REVIEW_TIME_PERIOD)`

After that change it's not going to be clear what the unit is, so can you leave a comment saying that it's hours? Or find a name for the var that includes the fact it's hours?

Reply:
Interesting! Ok will make that change.